### ISSUE TITLE
Move model chosen flag to internal property.

### ISSUE THREAD

**Issue by jgornick** [brian-mann/Backbone.Chooser#3]
Because Backbone.Chooser uses a model attribute to store the chosen state, when saving the model, the chosen flag is also sent in the request. 

In my scenario, the chosen field is not part of the original data that populated the model. My API doesn't recognize it in the schema, which causes an error.

Would it be possible to set the flag on a "private" property for the model?

**Comment by brian-mann** [comment]
Yes, this makes sense.  I should override the model's sync and slice the chosen attribute out so it's not sent to the backend.
